engine-tests/src/test/java/org/terasology/math/Diamond3iIteratorTest.java

```diff
@@ -49,9 +49,9 @@ public void testOneDistanceIteration() {
         assertTrue("Missing: " + expected, expected.isEmpty());
     }
 
-    @Test
+    /*@Test
     public void testTwoDistanceIteration() {
-        Set<Vector3i> iter = Sets.newHashSet(Diamond3iIterator.iterate(Vector3i.zero(), 2));
```

Comment:
Would suggest making an alternative zero constant rather than ```new```ing a vector for this.